### PR TITLE
Feature/customizable currency

### PR DESCRIPTION
**Pull request by rimrakhimov**
The request adds a new configuration variable _eth1Currency_ where the name of the currency used in depositing may be placed. The work is still in progress. There are several changes occurring on different pages:

1. The currency name located near login becomes customizable (however, another problem arises with absent currency symbol image).
    image

2. The staking token symbol in the validator page (`http://u.r.l./validator/{index}#deposits`) is as well changed on the configured value (although, it seems that some problems arise with "Validator history" block).
    image

The problems that reveal in the solution seem to be connected with the fact, that the structure with all currencies is hardcoded inside the golang code. For now I am analyzing this connection and think how it may be overcome. Thinking about replacing the first value in the structure from _ETH_ to the configurable value.

### CODE REVIEW

**Comment by rimrakhimov** [comment]
The first problem with absent currency symbol may be solved in the following way: we need to put the currency symbol in _.svg_ format (480x640) as `static/img/{eth1Currency}.svg`. In that case it will automatically update the symbol in accordance with a currency specified in the configuration file. No more changes inside the source code are required.

Additionally, I may suggest to completely remove the symbol if no image was found for the currency (or to use some default image). That may require some minor changes inside the HTML template.

**Comment by akolotov** [comment]
Could you summarize which originally opened issues this PR addresses?

**Comment by rimrakhimov** [comment]
The request adds a new configuration variable _eth1Currency_ where the name of the currency used in depositing may be placed. Furthermore, _eth1CurrencyName_ and _eth1CurrencyFormatDecimals_ have been added to fix representability problems. The request addresses several issues:

1. Closes the issue #13 by making the staking token name to be set by _eth1Currency_ configuration variable for the "Epochs" page.
    image

1. Closes the issue #20 by making the staking token symbol to be set by _eth1Currency_ configuration variable for the page displaying an epoch details.
    image

1. Closes the issue #23 by making the staking token symbol to be set by _eth1Currency_ configuration variable for the validator page. _eth1CurrencyFormatDecimals_ configuration variable is responsible for the number of digits to be shown in the fractional part of the numbers.
    image

1. Partially closes the issue #24 by making the staking token symbol on the deposit dashboard configurable to be set by _eth1Currency_ configuration variable. ("Eth1 Deposits" and "Eth2 Deposits" labels should be configured to close the issue)
    image

1. Partially closes the issue #4 by replacing ETH staking token symbol and name at the currency list with _eth1Currency_ and _eth1CurrencyName_ values. The solution requires the .svg image (640x480) to be placed in the format `eth1Currency.svg` in the `static/img/` directory. (ETH has to be still returned to close the issue).
    image

1. Partially closes the issue #9 by modifying staking token symbol on the "Validators/Overview" and "Validators/Leaderboard" pages (there is still balance column with ETH symbols left).
    image
    image

1. The request addresses staking token symbol on the "Validators/Deposits Leaderboard" page that is related to the issue #10 (still "Eth1 Address" values to be replaced).
    image

**Review by k1rill-fedoseev — APPROVED** [review]
Thanks! PR looks ok to me, let's merge it as is and then work on the remaining labels under a separate one.
Apart from what is already fixed, I see that we need to update the following ones:
* #6
* #14 
* Replace all remaining Eth1 and Eth2 labels.

Let's start working on these issues under a separate PR.
